src/ssa/parse.js

```diff
@@ -69,7 +70,7 @@ function parseSsa(data) {
 module.exports = {
```

Comment:
SSA can do same exports as SRT, but lets leave that until the module is broken out on its own